packages/block-library/src/search/view.js

```diff
@@ -0,0 +1,61 @@
+window.addEventListener( 'DOMContentLoaded', () => {
```

Comment:
I wonder if this could use the new interactivity API cc @gziolo

Reply:
Yes, it definitely could. We will run the experiment for Interactivity API until everything is properly tested so it's fine to have two concurrent solutions for 2-3 Gutenberg plugin releases 👍🏻